nikola/__main__.py

```diff
@@ -241,14 +241,15 @@ def load_tasks(self, cmd, opt_values, pos_args):
         if self.quiet:
             DOIT_CONFIG = {
                 'verbosity': 0,
-                'reporter': 'zero',
+                'reporter': 'zero'
```

Comment:
(It also lets the next person just type in their changes without having to care about adding a comma.)